src/FieldEncryptProcessEntities.php

```diff
@@ -0,0 +1,302 @@
+<?php
+/**
+ * @file Contains \Drupal\field_encrypt\FieldEncryptProcessEntities
+ */
+
+namespace Drupal\field_encrypt;
+
+use Drupal\Core\Entity\EntityManager;
+use Drupal\Core\Entity\Query\QueryFactory;
+use Drupal\encrypt\EncryptService;
+
+/**
+ *
+ */
+class FieldEncryptProcessEntities {
+
+  /**
+   * Filter (blacklist) field types that don't work with encryption.
+   *
+   * This list is mostly to avoid problems, but we shouldn't let people encrypt
+   * these fields since we know it wouldn't work.
+   */
+  public $blacklist_fields = [
+    'image',
```

Comment:
I sort of think we need to look at a plugin system that registers specific field types (in lieu of a black listing system). This plugin manager could then  be used for something like "File Encrypt" for things like image, etc